mappings/net/minecraft/util/Util.mapping

```diff
@@ -259,17 +312,37 @@ CLASS net/minecraft/class_156 net/minecraft/util/Util
 			COMMENT a function that, when given the size of {@code values},
 			COMMENT returns a map for storing the indices of the values
 	METHOD method_44893 getFormattedCurrentTime ()Ljava/lang/String;
+		COMMENT {@return the current time formatted using {@link #DATE_TIME_FORMATTER}}
 	METHOD method_645 previous (Ljava/lang/Iterable;Ljava/lang/Object;)Ljava/lang/Object;
+		COMMENT {@return the item preceding {@code object} in {@code iterable}}
+		COMMENT
+		COMMENT @implNote If {@code object} is not in {@code iterable}, this enters into an infinite loop.
```

Comment:
Unlike `next`, `previous` does not enter into an infinite loop.

I think these methods *expect* `object` to be in `iterable`. (If that's the case, these methods should throw `IllegalArgumentException` though.)